google-ads-examples/src/main/java/com/google/ads/googleads/examples/reporting/ParallelReportDownload.java

```diff
@@ -186,26 +191,25 @@ public void onStart(StreamController controller) {
 
     @Override
     public void onResponse(SearchGoogleAdsStreamResponse response) {
-      // Does something useful with the row. In this case we just count the
-      // records. Note that this method may be called from multiple threads, though always in
-      // sequence.
-      numRecords.incrementAndGet();
+      // Does something useful with the response. In this case we just count the responses.
+      // Note: this method may be called from multiple threads, though always in sequence.
```

Comment:
Took me a minute to understand "Though always in sequence". Consider rephrasing to clarify that you mean that responses passed here will be in the order returned by the API (I _think_ that's what you're trying to get across here).